.github/workflows/rust_ci.yml

```diff
@@ -30,9 +30,6 @@ jobs:
           sharedKey: base
           working-directory: ironfish-rust
 
-      - name: Check for license headers
```

Comment:
Just re-ordering these to be with the fmt and clippy, so that tests will run even if this fails, making the CI run a little more useful